### DISCUSSION TITLE
Webshare.io - Verification and Activity Meter issues

### DISCUSSION THREAD

**Discussion by KevinBytesTheDust** [KevinBytesTheDust/CVAmp#294] [General]
I’m trying to understand how many users are affected by the activity meter and the need for verification, or are simply happy with Webshare. If you’ve encountered this, please share your experience, including why you ran into the verification, which URLs were troublesome, and whether you passed the verification. This will help me determine if I can still recommend webshare.

**Comment by Alexistexas9697** [comment]
I use webshare, the price is good and I did the verification. I don't have any problems with verification right now. However, it detects all my tokenized accounts as "bot detected" and therefore I can't raid. I use a "Datacenter - Shared" proxy. Because I want the proxies to come from Turkey. Turkish proxies are only available in the "datacenter-shared" section and their prices are reasonable. However, I can't "raid" with this, this is very annoying. Static proxies don't have TR and their prices are extremely expensive (my monthly income doesn't cover this :D) Our biggest problem right now is that the raid feature can't be used easily. A solid proxy site where we can use the "Raid" feature (with Turkey proxies) and where the monthly payment is cheap would be very useful for us. It would be great if you focus on making the "Raid" feature easier.

**Reply by KevinBytesTheDust** [reply]
Yes, the purely residential proxies are not recommendable for this use case. I am open for ideas for other proxy providers, which I can thoroughly test.

**Reply by Alexistexas9697** [reply]
Despite my extensive research, I unfortunately could not find another proxy provider that works properly. If someone who knows can help me, I would be very happy. However, I definitely need a Turkey proxy.

**Reply by KevinBytesTheDust** [reply]
Same here 🤷‍♀️

**Reply by Darkestangel69** [reply]
> I use webshare, the price is good and I did the verification. I don't have any problems with verification right now. However, it detects all my tokenized accounts as "bot detected" and therefore I can't raid. I use a "Datacenter - Shared" proxy. Because I want the proxies to come from Turkey. Turkish proxies are only available in the "datacenter-shared" section and their prices are reasonable. However, I can't "raid" with this, this is very annoying. Static proxies don't have TR and their prices are extremely expensive (my monthly income doesn't cover this :D) Our biggest problem right now is that the raid feature can't be used easily. A solid proxy site where we can use the "Raid" feature (with Turkey proxies) and where the monthly payment is cheap would be very useful for us. It would be great if you focus on making the "Raid" feature easier.

How you get tokens? Making 100 Emails just to get different tokens manual sucks, DM me please

**Comment by noxwinslol11** [comment]


**Reply by KevinBytesTheDust** [reply]
Sorry to hear that. Does webshare give your the option to verify your account? After verification, you should be able to use your account as usual.

**Comment by dedalo88888** [comment]
Good morning,

WebShare changed the format of the Proxies, now they are like this:
45.43.XX.64:XXXX:XXXXXXX:XXXXXXX
before they were like this:
45.43.XX.64:XXXX:XXXXXXX
I am using your program CVAmp_KOFI_v14_CHROME129b_0.8.0_20240928_Windows-X64

I need help, nothing works

**Reply by dedalo88888** [reply]
I managed to download it as before and CVAMP detects the proxies but I keep dying because:
TWITCH Instance 1 died: Error:Page.goto: net::ERR_TUNNE... Please see cvamp.log.
It doesn't work with twitch, kick or Youtube

**Reply by dedalo88888** [reply]
It's already fixed.

It was a WebShare problem where they removed my proxies without permission, according to them due to a computer error.
